### PR TITLE
[changelog] start changelog for releases

### PR DESCRIPTION
**Pull request by sausagee**
## Motivation
Start changelog for next and future releases.

Also add an ignore rule in x for documentation changes.

## Test Plan
No test other than CI

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Changelog
+==========================
+This doc describes all notable changes to Diem Core and Diem Framework between.
```
Comment (sherry-x) [inline CHANGELOG.md:3]: this sentence seems either not complete or mean something else? :P

Comment (sausagee) [reply CHANGELOG.md:3]: copy pasta from quip

**Review by sherry-x — APPROVED** [review]

**Review by dahliamalkhi — APPROVED** [review]
LGTM, thanks for establishing this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Changelog
```
Comment (mgorven) [inline CHANGELOG.md:1]: Formatting nit: You can't have both `#` prefix and `===` underline.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+*[Changed]*
+
+* Rename mint to faucet container for readability in testnet. (#7618)
+* Use branch specific CI container (see #7649)
+* Backup & restore tooling to accept genesis or latest waypoint (#7465)
+
+*[Security]*
+
+* Add RUSTSEC-2021-0020 to audit ignore list (#7723)
```
Comment (frebours) [inline CHANGELOG.md:20]: Can you elaborate please? From what I understand, the RUSTSEC-2021-0020 is a vulnerability and we choose to ignore it. Why don't we upgrade to a patched version as define here https://rustsec.org/advisories/RUSTSEC-2021-0020.html? Sorry if my question is stupid.

Comment (sherry-x) [reply CHANGELOG.md:20]: @frebours Young put more details in the PR which made this change https://github.com/diem/diem/pull/7723

**Review by frebours — APPROVED** [review]
I don't see any blocker.

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## [diem-core-v1.1.5] - 2021-03-02
+*[Added]*
+
+* Add NetworkIdentity to metrics dashboard (#7669)
+* Add health check of node status via json-rpc (#7498)
+* Add genesis smoke test to backup & restore tooling (#7465)
+
+*[Changed]*
+
```
Comment (pdhamdhere) [inline CHANGELOG.md:13]: We should probably add "Increase max number of child accounts to 2^16 form 2^8"

Comment (sherry-x) [reply CHANGELOG.md:13]: that change is not in this release, see the diff view https://github.com/diem/diem/compare/diem-core-v1.1.4...diem-core-v1.1.5

the child account size change went in pretty early, before we spin up Mainnet

**Comment by sherry-x** [comment]
@sausagee should we propagate this changelog file on main branch too? since we're going to ask people to add major changes into the changelog going forward

**Pull request merged**